frontend/src/modules/SimulationTimeSeriesMatrix/utils/ensemblesContinuousParameterColoring.ts

```diff
@@ -12,22 +13,20 @@ export class EnsemblesContinuousParameterColoring {
      */
 
     private _parameterIdent: ParameterIdent;
-    private _ensembleContinuousParameterSet: { [ensembleName: string]: ContinuousParameter };
+    private _ensembleContinuousParameterSet: Map<EnsembleIdent, ContinuousParameter>;
     private _colorScale: ColorScale;
 
     constructor(selectedEnsembles: Ensemble[], parameterIdent: ParameterIdent, colorScale: ColorScale) {
         this._parameterIdent = parameterIdent;
-        this._ensembleContinuousParameterSet = {};
+        this._ensembleContinuousParameterSet = new Map<EnsembleIdent, ContinuousParameter>();
```

Comment:
Should this be called `_ensembleContinuousParameterMap`?

Reply:
👍